portfolio.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Edie's Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+  </head>
+  <body>
+    <nav>
+      <a href= "#"> Home</a> | <a href="#">Portfolio</a>
+    </nav>
+    <h1>
+      Edie Windsor
+    </h1>
+    <h2 class="experience">
+      Experience 
+    </h2>
+    <img class="edie2" src="images/edie2.jpg">
```

Comment:
@akallbold This image is not loading because the file the file is called `Edie2.jpg` and you are requesting `images/edie2.jpg` 🔨